### PR TITLE
Martin/jib 181 show two messages in the message cell

### PR DESCRIPTION
**Pull request by MartinJibber**

### CODE REVIEW

File: Jibber.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -12,7 +12,7 @@
 		7F227B8576E160C9469E08D1 /* Pods_Intents_Extension.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 2C50C41C9E4C24DF432E0D86 /* Pods_Intents_Extension.framework */; };
 		8007CB642703CE0600FEA511 /* AttachmentCollectionViewDataSource.swift in Sources */ = {isa = PBXBuildFile; fileRef = 8007CB632703CE0600FEA511 /* AttachmentCollectionViewDataSource.swift */; };
 		800FAAC126F0585A00D3C3A0 /* Conversation+Extensions.swift in Sources */ = {isa = PBXBuildFile; fileRef = 800FAAC026F0585A00D3C3A0 /* Conversation+Extensions.swift */; };
-		8015CA7726F3D907000F3C60 /* MessageCell.swift in Sources */ = {isa = PBXBuildFile; fileRef = 8015CA7626F3D907000F3C60 /* MessageCell.swift */; };
+		8015CA7726F3D907000F3C60 /* ConversationMessageCell.swift in Sources */ = {isa = PBXBuildFile; fileRef = 8015CA7626F3D907000F3C60 /* ConversationMessageCell.swift */; };
```
Comment (MartinJibber) [inline Jibber.xcodeproj/project.pbxproj:15]: Message cell was too ambiguous. This name specifies the context that the cell is used in.

File: Shared/UI/Views/TextView.swift [hunk 2]
```diff
@@ -160,7 +160,8 @@ class TextView: UITextView {
                                             longestEffectiveRange: nil,
                                             in: NSRange(location: 0, length: attText.length))
 
-        let maxSize = CGSize(width: width, height: height)
+        let maxSize = CGSize(width: width - self.textContainerInset.left - self.textContainerInset.right,
+                             height: height - self.textContainerInset.top - self.textContainerInset.bottom)
```
Comment (MartinJibber) [inline Shared/UI/Views/TextView.swift:164]: Improving get size calculation on TextView.

File: iOS/Flows/Conversation/Cells/ConversationMessageCell.swift [hunk 3]
```diff
@@ -36,17 +35,21 @@ class MessageCell: UICollectionViewCell {
     private let headerRegistration
     = UICollectionView.SupplementaryRegistration<TimeSentView>(elementKind: UICollectionView.elementKindSectionHeader) { supplementaryView, elementKind, indexPath in
     }
+    private let footerRegistration
+    = UICollectionView.SupplementaryRegistration<TimeSentView>(elementKind: UICollectionView.elementKindSectionFooter) { supplementaryView, elementKind, indexPath in
+    }
```
Comment (MartinJibber) [inline iOS/Flows/Conversation/Cells/ConversationMessageCell.swift:39]: To show dates for messages the user has sent.

File: iOS/Flows/Conversation/Cells/ConversationMessageCell.swift [hunk 4]
```diff
@@ -10,11 +10,9 @@ import Foundation
...
 import UIKit
 
-
-
-/// A cell to display a message along with a limited number of replies to that message.
-/// The root message and replies are stacked along the z-axis, with the most recent reply at the front (visually obscuring the others).
-class MessageCell: UICollectionViewCell {
+/// A cell to display a high-level view of a conversation's message. Displays a limited number of recent replies to the message.
+/// The user's replies and other replies are put in two stacks (along the z-axis), with the most recent reply at the front (visually obscuring the others).
+class ConversationMessageCell: UICollectionViewCell {
```
Comment (MartinJibber) [inline iOS/Flows/Conversation/Cells/ConversationMessageCell.swift:15]: Keeping the documentation fresh.

**Review by Benjamindodgson — APPROVED** [review]

**Pull request merged**